src/tespy/components/heat_exchangers.py

```diff
@@ -1263,6 +1283,27 @@ def calc_parameters(self):
 
         self.check_parameter_bounds()
 
+    def exergy_balance(self, Tamb):
+        r"""
+        Perform exergy balance of a parabolic trough.
+
+        Tamb : float
+            Ambient temperature Tamb / K.
+
+            .. math::
+
+                Ex_{output} = \dot{m}_{in} \cdot (ex_{ph,out} - ex_{ph,in})
+                Ex_{input} = \dot{Q}_{sol} \cdot \left(1 - \frac{4}
+                                                       {3} \frac{T_{amb}}
+                                                       {T_{sun}}\right)
+                Ex_{loss} = 0
+        """
+        T_sun = 5679
+        self.Ex_output = self.inl[0].m.val_SI * (self.outl[0].ex_physical
+                                                 - self.inl[0].ex_physical)
+        self.Ex_input = self.Q.val * (1 - ((4/3) * (Tamb/T_sun)))
+        self.Ex_loss = 0
```

Comment:
- linebreak after parenthesis
- missing spaces around 4 / 3 and Tamb / Tsun
- similar issue with the exergy balance here. Although Q should only be > 0, there might be exergy losses due to pressure losses we do not yet consider.